tests/integration/targets/ec2_instance/roles/ec2_instance/meta/main.yml

```diff
@@ -0,0 +1,5 @@
+dependencies:
+  - prepare_tests
```

Comment:
`prepare_tests` includes an empty file. Why do we need it?

Reply:
TBH that's been migrated around as-is since it was in ansible/ansible.  It would probably we good to do a general review but I'd prefer not to make too many changes here beyond what's needed to get the modules working